docs/src/dictvectors.md

```diff
@@ -35,11 +35,9 @@ iteration on them is disabled by default. Iteration must be explicitly performed
 similar functions when performing reductions, as that will make the operations compatible
 with MPI.
```

Comment:
`VectorInterfaces.jl` could also be explicitly mentioned in the docs here, since you are discussing vector operations here, not just the module docstrings.

Also, typo on `values`.